roles/astro-gcp-role-api-service-agent.yaml

```diff
@@ -0,0 +1,387 @@
+- container.secrets.get
+- container.secrets.list
+- container.secrets.update
+- container.serviceAccounts.create
+- container.serviceAccounts.createToken
+- container.serviceAccounts.delete
+- container.serviceAccounts.get
+- container.serviceAccounts.list
+- container.serviceAccounts.update
+- container.services.create
+- container.services.delete
+- container.services.get
+- container.services.getStatus
+- container.services.list
+- container.services.proxy
+- container.services.update
+- container.services.updateStatus
+- container.thirdPartyObjects.create
+- container.thirdPartyObjects.delete
+- container.thirdPartyObjects.get
+- container.thirdPartyObjects.list
+- container.thirdPartyObjects.update
+- container.validatingWebhookConfigurations.create
+- container.validatingWebhookConfigurations.delete
+- container.validatingWebhookConfigurations.get
+- container.validatingWebhookConfigurations.list
+- container.validatingWebhookConfigurations.update
+- deploymentmanager.deployments.cancelPreview
+- deploymentmanager.deployments.create
+- deploymentmanager.deployments.delete
+- deploymentmanager.deployments.get
+- deploymentmanager.deployments.list
+- deploymentmanager.deployments.stop
+- deploymentmanager.deployments.update
+- deploymentmanager.manifests.get
+- deploymentmanager.manifests.list
+- deploymentmanager.operations.get
+- deploymentmanager.operations.list
+- deploymentmanager.resources.get
+- deploymentmanager.resources.list
+- deploymentmanager.typeProviders.create
+- deploymentmanager.typeProviders.delete
+- deploymentmanager.typeProviders.get
+- deploymentmanager.typeProviders.getType
+- deploymentmanager.typeProviders.list
+- deploymentmanager.typeProviders.listTypes
+- deploymentmanager.typeProviders.update
+- deploymentmanager.types.create
+- deploymentmanager.types.delete
+- deploymentmanager.types.get
+- deploymentmanager.types.list
+- deploymentmanager.types.update
+- dns.changes.create
+- dns.changes.get
+- dns.managedZones.create
+- dns.managedZones.delete
+- dns.managedZones.get
+- dns.managedZones.list
+- dns.managedZones.update
+- dns.resourceRecordSets.create
+- dns.resourceRecordSets.delete
+- dns.resourceRecordSets.list
+- iam.roles.create
+- iam.roles.delete
+- iam.roles.get
+- iam.roles.list
+- iam.roles.update
+- logging.buckets.get
+- logging.buckets.list
+- logging.exclusions.get
+- logging.exclusions.list
+- logging.links.get
+- logging.links.list
+- logging.locations.get
+- logging.locations.list
+- logging.logEntries.list
+- logging.logMetrics.get
+- logging.logMetrics.list
+- logging.logServiceIndexes.list
+- logging.logServices.list
+- logging.logs.list
+- logging.operations.get
+- logging.operations.list
+- logging.queries.create
+- logging.queries.delete
+- logging.queries.get
+- logging.queries.list
+- logging.queries.listShared
+- logging.queries.update
+- logging.sinks.get
+- logging.sinks.list
+- logging.usage.get
+- logging.views.get
+- logging.views.list
+- resourcemanager.projects.get
```

Comment:
i believe we also need `resourcemanager.projects.get`. pinged you on a slack thread

Reply:
Added back in 👍